powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/PrepareActivationRequest.java

```diff
@@ -29,16 +31,41 @@
 @Data
 public class PrepareActivationRequest {
 
+    @Schema(description = "Activation code")
+    @NotBlank(message = "Activation code must not be empty when preparing activation")
     private String activationCode;
+
+    @Schema(description = "Application key")
+    @NotBlank(message = "Application key must not be empty when preparing activation")
     private String applicationKey;
+
+    @Schema(description = "Whether recovery codes should be generated")
     private boolean generateRecoveryCodes;
+
+    @Schema(description = "Identifier of the temporary key for encryption")
     private String temporaryKeyId;
+
+    @Schema(description = "Ephemeral public key used in encryption")
+    @NotBlank(message = "Ephemeral public key must not be empty when preparing activation")
     private String ephemeralPublicKey;
+
+    @Schema(description = "Encrypted data used in encryption")
```

Comment:
The second part of the sentence does not sound right.
```suggestion
    @Schema(description = "Encrypted data")
```